test/functional/http/post/common.js

```diff
@@ -110,12 +110,23 @@ function invalidAssets(option, badTransactions) {
 					badTransactions.push(transaction);
 				});
 		});
-	});
 
-	describe(`using invalid asset.${option} values`, () => {
-		typesRepresentatives.allTypes.forEach(test => {
-			it(`using ${test.description} should fail`, () => {
-				transaction.asset[option] = test.input;
+		describe(`using invalid asset.${option} values`, () => {
```

Comment:
This nests this `describe` block in the previous one. The proposed solution was to create an additional `describe` block which encompasses both. Does nesting make sense?

Reply:
Yeah, I am gonna change the suite to have like this.
```
describe('Using invalid asset values', () => {
	describe('without option', () => {});
	describe('with option', () => {});
});
```